contrib/example-backend/http/handler.go

```diff
@@ -75,6 +77,7 @@ func NewHandler(
 	scope kubebindv1alpha1.Scope,
 	mgr *kubernetes.Manager,
 	apiextensionsLister apiextensionslisters.CustomResourceDefinitionLister,
+	oidcAuthorizeURL string,
```

Comment:
why not keep the order? Put it behind the callback URL everywhere and name both similarly.